### PR TITLE
Audio feature extractor

### PR DESCRIPTION
**Pull request by aiden200**

### CODE REVIEW

File: preprocess/extract_features.py [hunk 1]
```diff
@@ -107,12 +107,19 @@ def extract_audio(audio_path, audio_model, n_feats):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser("Dataset Feature Extraction")
+    parser.add_argument("--data_dir", type=str)
     parser.add_argument("--video_backbone", type=str)
     parser.add_argument("--audio_backbone", type=str, default="MFCC")
-    parser.add_argument("--root_dir", default="2D3MF_Datasets")
-    parser.add_argument("--dataset", default="Forensics++", type=str)
+    parser.add_argument("--Forensics", action="store_true")
```
Comment (aromanusc) [inline preprocess/extract_features.py:113]: why is Forensics an option here?

File: requirements2.txt
Comment (aromanusc) [inline requirements2.txt:1]: is this file needed/used?

File: preprocess/preprocess_clips.py
Comment (aromanusc) [inline preprocess/preprocess_clips.py:1]: Given that our video and audio filenames are the same filenames for both, I think face cropping and feature extraction can be handled with the same code for all dataset. However, we do need to handle the train/validation/test splitting in a different way for each dataset. But that happens are we have gotten the face cropping, marlin embeddings, and audio emneddings.

This means that the Faceforensics logic here may not be needed as it adds extra complexity to the feature extraction. Perhaps we should start a new set of files called `generate_splits.py`, which should handle the each of our dataset splitting for their corresponding features.

File: requirements2.txt
Comment (aiden200) [reply requirements2.txt:1]: Yes, that is the testing of requirements.txt

**Review by aromanusc — APPROVED** [review]
